crypto/kzg/src/lib.rs

```diff
@@ -36,20 +44,32 @@ impl From<c_kzg::Error> for Error {
 #[derive(Debug)]
 pub struct Kzg {
     trusted_setup: KzgSettings,
+    context: PeerDASContext,
 }
 
 #[automock]
 impl Kzg {
     /// Load the kzg trusted setup parameters from a vec of G1 and G2 points.
     pub fn new_from_trusted_setup(trusted_setup: TrustedSetup) -> Result<Self, Error> {
+
+        // Initialize the trusted setup using default parameters
+        // 
+        // Note: One can also use `from_json` to initialize it from the consensus-specs
+        // json string.
+        let peerdas_trusted_setup = PeerDASTrustedSetup::default();
+        // Set the number of threads to be used
+        //
+        // we set it to 1 to match the c-kzg performance
+        const NUM_THREADS : usize = 1;
+
+        let context = PeerDASContext::with_threads(&peerdas_trusted_setup, NUM_THREADS);
+
         Ok(Self {
             trusted_setup: KzgSettings::load_trusted_setup(
                 &trusted_setup.g1_points(),
                 &trusted_setup.g2_points(),
-                // Enable precomputed table for 8 bits, with 96MB of memory overhead per process
-                // Ref: https://notes.ethereum.org/@jtraglia/windowed_multiplications
-                8,
```

Comment:
This parameter does not exist on the c-kzg 4844 auditted branch -- since I've reverted c-kzg back to v1, it has been removed